bulk/repo-access-api.adoc

```diff
@@ -5,11 +5,12 @@ include::../shared/issue-footnotes.adoc[]
 :arch: ../reference-architecture/reference-architecture
 :chunk: <<{serialization}.adoc#SerializationChunk, SerializationChunk>>
 
+[[bulk]]
 = Bulk Repository Access API
 :toc: preamble
 :toclevels: 2
 
-The bulk API is used to store and retrieve nodes in batches at the moment of invocation.{fn-org25}
+The bulk API *version 2024.1* is used to store and retrieve nodes in batches at the moment of invocation.{fn-org25}
```

Comment:
Can the user specify that they want to use a different version of the API or is the latest one always used?

Reply:
Not decided yet. We possibly change future versions of the spec, so I want to make sure the document contains the version.